extensions/math/init.lua

```diff
@@ -1,8 +1,66 @@
 --- === hs.math ===
 ---
 --- Various helpful mathematical functions
+---
+--- This module will also mirror the contents of the built-in Lua `math` library so it is safe to do something like the following in your own code and still have access to both libraries:
```

Comment:
How about we change "will also mirror" to "includes, and is a superset of"? It moves it into a more active voice and is explicit that this is Lua math + more stuff.